.drone.yml

```diff
@@ -17,6 +17,10 @@ steps:
         from_secret: NPM_AUTH
       NPM_EMAIL:
         from_secret: NPM_EMAIL
+      REALM_API_URL:
+        from_secret: REALM_API_URL
+      REALM_SEARCH_URL:
```

Comment:
For search, I didn't think we needed to include here, but this is fine. I don't think you need a secret key for this one.

Reply:
Ahh ok, I just figured we could keep our realm URLs obscured across the board with this.